site/content/docs/5.0/components/skeletons.md

```diff
@@ -0,0 +1,147 @@
+        </span>
+        <span class="skeleton-line skeleton-glow skeleton-sm">
+          <span class="skeleton-dash bg-secondary" style="width: 70%;"></span>
+        </span>
+      </p>
+      <a href="#" class="btn btn-primary disabled" style="width: 55%;">
+        <span class="skeleton-line w-100"></span>
+      </a>
+    </div>
+  </div>
+</div>
+{{< /example >}}
+
+## How it works
+
+Skeletons are composed of `.skeleton-line`s. Each of them needs one or more `.skeleton-dash`es.
+
+{{< example >}}
+<span class="skeleton-line">
+  <span class="skeleton-dash"></span>
+</span>
+{{< /example >}}
+
+{{% callout info %}}
+Beside of using `aria-busy="true"` to indicate the subtree is being updated, if both, skeleton and the populated element, are present on the content, `aria-hidden="true"` should be used to hide one or another from screen readers.
+{{% /callout %}}
+
+
+### Width
+
+By default all `.skeleton-dash`es together will take the full with of the `.skeleton-line`. Still, you can customize them with custom css widths or even using the utility classes.
+
+{{< example >}}
+<span class="skeleton-line">
+  <span class="skeleton-dash"></span>
+  <span class="skeleton-dash"></span>
+</span>
+<span class="skeleton-line">
+  <span class="skeleton-dash w-25"></span>
+  <span class="skeleton-dash w-75"></span>
+</span>
+<span class="skeleton-line">
+  <span class="skeleton-dash" style="width: 60%;"></span>
+  <span class="skeleton-dash" style="width: 25%;"></span>
+</span>
+{{< /example >}}
+
+### Color
+
+By default the `skeleton-dash` use the `currentColor`. Again, you can override it with a custom color or utitlity class.
+
+{{< example >}}
+<span class="skeleton-line">
+  <span class="skeleton-dash"></span>
+</span>
+<span class="skeleton-line">
+  <span class="skeleton-dash bg-primary"></span>
+  <span class="skeleton-dash bg-secondary"></span>
+  <span class="skeleton-dash bg-success"></span>
+  <span class="skeleton-dash bg-danger"></span>
+  <span class="skeleton-dash bg-warning"></span>
+  <span class="skeleton-dash bg-info"></span>
+  <span class="skeleton-dash bg-light"></span>
+  <span class="skeleton-dash bg-dark"></span>
+  <span class="skeleton-dash bg-white"></span>
+  <span class="skeleton-dash bg-transparent"></span>
+</span>
+{{< /example >}}
+
+### Sizing
+
+The size of the `.skeleton-dash`es are calculated according to the typography style of the parent element. To customize them you can use classes like `.skeleton-lg`, `.skeleton-sm` or `.skeleton-xs`.
+
+{{% callout info %}}
+Even if you use size modifiers, the space between lines is not affected, only the thickness of the dash. This is in purpose to help you customize the weights of the different elements of the skeleton composition.
+{{% /callout %}}
+
+{{< example >}}
+<div class="d-flex justify-content-around">
+  <div style="width: 45%;">
+    <h3>
+      <span class="skeleton-line"><span class="skeleton-dash"></span>&#8232;</span>
```

Comment:
Not a huge fan of one nested one-liners. If it doesn't affect the display, I'd say better nest the elements for better readability.

Reply:
Should I inline the h3 with the skeleton classes then? Is that what you are saying? 🤔

Reply:
```html
<span class="skeleton-line">
  <span class="skeleton-dash"></span>
  &#8232;
</span>
```

But I don't think the `&#8232;` solution is right. I wait for @patrickhlauke's feedback about that.